src/main/scala/firrtl/AddDescriptionNodes.scala

```diff
@@ -28,6 +28,13 @@ case class DocStringAnnotation(target: Target, description: String) extends Desc
       case Some(seq) => seq.map(n => this.copy(target = n))
     }
   }
+  override private[firrtl] def dedup: Option[(Any, Annotation, ReferenceTarget)] = this match {
+    case a @ DocStringAnnotation(refTarget: ReferenceTarget, _) =>
+      Some(((refTarget.pathlessTarget, description), copy(target = refTarget.pathlessTarget), refTarget))
+    case a @ DocStringAnnotation(pathTarget: ReferenceTarget, _) =>
+      Some(((pathTarget.pathlessTarget, description), copy(target = pathTarget.pathlessTarget), pathTarget))
```

Comment:
Unless I'm missing something, these both match (and do) the same thing.
```suggestion
    case DocStringAnnotation(refTarget: ReferenceTarget, _) =>
      Some(((refTarget.pathlessTarget, description), copy(target = refTarget.pathlessTarget), refTarget))
```
Also no need to bind `a` since you have `this`